CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -652,6 +721,13 @@ SiPixelLAHarvest::fitResults SiPixelLorentzAnglePCLHarvester::fitAndStore(
   hists_.h_bySectChi2_->setBinContent(i_index, res.redChi2);
```

Comment:
line 713: `double errsq_LA = ...`
line 720: `hists_.h_bySectMeasLA_->setBinError(i_index, (res.error_LA/ theMagField_));`

`res.error_LA` was calculated with error propagation but assuming no correlations between fit parameters. Would it be better to replace this with `sqrt(error_f1(half_width) + error_f1(0))/half_width = sqrt(res.dSq + CovMatrix(0, 0))/half_width`?

Reply:
@wweiphy why do you assume `error_f1(0)` = `CovMatrix(0,0)` ?

Reply:
Because the rest of the terms become 0 if `d = 0`
[delta_d^2_=_sum_1.pdf](https://github.com/cms-sw/cmssw/files/9337811/delta_d.2_._sum_1.pdf)

Reply:
posting the formula so that it gets readily readable:
![Screenshot from 2022-08-15 16-40-03](https://user-images.githubusercontent.com/5082376/184656611-9875a7d1-2881-4258-ae04-de42557caa2e.png)

Reply:
@wweiphy done at [490e6ae](https://github.com/cms-sw/cmssw/pull/38996/commits/490e6ae7e493e99ddb5a9259f92c65aa0d4120d4)